addons/accessibility/fnc_vanillaSoundPlayed.sqf

```diff
@@ -0,0 +1,74 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_accessibility_fnc_vanillaSoundPlayed
+
+Description:
+    Handles muting and hinting of vanilla sounds
+
+Parameters:
+    _unit - unit that played the sound <OBJECT>
+    _sound - sound that was played <NUMBER>
+
+Returns:
+    0 if the sound should be muted <NUMBER>
+
+Author:
+    Brett Mayson
+---------------------------------------------------------------------------- */
+
+params [
+    ["_unit", objNull, [objNull]],
+    ["_sound", 0, [0]]
+];
+
+if (_unit == objNull) exitWith {false};
```

Comment:
isNull _unit

Reply:
null == null returns false.

Reply:
Exactly. The check currently is a paradox.